### ISSUE TITLE
BioModel Discussion Issue for eos8fth

### ISSUE THREAD

**Issue by Zainab-ik** [ersilia-os/eos8fth#8]
**Summary !!!**

- **BioModel Name**: KC2021 -  A machine learning platform to estimate anti-SARS-CoV-2 activities

- **BioModel Tag**: fair AIML, Machine learning model, Ersilia

- **Metadata Comments**: 
1. This model is similar to Zairachem model, right?

- **Contributor**:
> 
 | roles             | Names | 
 | :---------------- | :------: | 
 | Curator        |   @Zainab-ik   | 
 | Code contributor      |   Pradnya2203      | 
 | Reviewer       |   @GemmaTuron    | 

- **Curation status** :  manually curated

- **Annotation file**: [here](https://docs.google.com/spreadsheets/d/17IqABILNm0tYtwIntXHwJLOm4ECRM_hE4x0CNXyZkoQ/edit?usp=drive_link)

**Comment by GemmaTuron** [comment]
Yes, we could consider that a similar approach as ZairaChem, good observation

**Comment by Zainab-ik** [comment]
- This model has multiple output from my understanding. I'd like to confirm if the similarity search feature and associated experimental data prediction feature is implemented in Ersilia? 
- For the model output; can _active_ and _inactive_ best classify it?
- They made mention of 3 data input format; drug name, PubChem compound ID number and SMILES format.Does all input format also works with the Ersilia model? The input query still convert the drugs and PubChem CID number to smiles. I think smiles would just be okay as input.
- 24 different classifiers were trained for REDIAL, i'm confused on how that'd fit into the metadata. can we go with just _classification models_ as the metadata. 
- There's no mention of cross-validation in this paper.
- The publication attached to this model ([this](https://www.nature.com/articles/s42256-021-00335-w#Abs1)) is not in PUBMED, I however found [this](https://pubmed.ncbi.nlm.nih.gov/33151287/). This is more about Drug Central and the authors are different. How do we reconcile both?

**Comment by Zainab-ik** [comment]
@GemmaTuron The model is ready for review.

**Comment by GemmaTuron** [comment]
Hi @Zainab-ik 

I do not understand why some target organisms are added as Taxon and others as Property. If you want to understand more about the output you can run the model from the web implementation even.
I do not understand this question, can you clarify? What is not clear about the classifiers 
_24 different classifiers were trained for REDIAL, i'm confused on how that'd fit into the metadata. can we go with just classification models as the metadata._
The publication is a Nature paper, so even if it is not in Pubmed, it is the right link to cite

In the methds they say:
_ll models were optimized and selected based on the validation F1 score._ so there must be some crossvalidation happening

**Comment by Zainab-ik** [comment]
- If a target organism doesn't have a taxonomy(taxon ID), for example, an enzyme like ACE2 doesn't have taxonomy and is classified as a gene, the right ontology would be a _Genetic Ontology:GO_ or _NCI Thesaurus OBO Edition NCIT:NCIT_, they are classified as properties.
- I meant if I should list the classifiers individually. I'd go with classification model as a metadata. 
- I added the publication as DOI in the metadata

**Comment by GemmaTuron** [comment]
Hi @Zainab-ik 

What is the In Vitro field? what does it refer to?
And the Active in Assay and Inactive in Assay? I had not seen them before

**Comment by Zainab-ik** [comment]
> Hi @Zainab-ik
> 
> What is the In Vitro field? what does it refer to? And the Active in Assay and Inactive in Assay? I had not seen them before

The invitro is for data generation and thus removed. 
The output of the model is either active in an assay or inactive in an assay. 
All redundant field removed.

**Comment by GemmaTuron** [comment]
Ok, wouldn't all model outputs be "active in an assay" or "inactive in an assay" 
how is this different from "Active" and "Inactive"? Maybe it is redundant?

**Comment by Zainab-ik** [comment]
> Ok, wouldn't all model outputs be "active in an assay" or "inactive in an assay" how is this different from "Active" and "Inactive"? Maybe it is redundant?

It's actually the same ontology term that it is annotated with, and the output was based on Redial-2020 prediction. 
I don't think it's redundant. It fits better as an output for this model rather than a property, whereas, it's a property that's general to all model. When a model predicts a compound, it's either active for a particular bioactivity or not. And for this model, It's an assay activity.

**Comment by GemmaTuron** [comment]
Hi Zainab,

I am not convinced of the difference, as this model provides a bunch of bioactivity predictions actually, I do not understand what you are referring to as "assay"
Let's mark this to discuss in our next meeting

**Comment by Zainab-ik** [comment]
Interpretation: The model returns the probability of 1 (active) in each assay. Good drugs are active in CPE, 3CL and are inactive in cytotox, hCYTOX and ACE2 and/or are active in at least one of the following: AlphaLISA, CoV-PPE, MERS-PPE, while inactive in the counter screen, respectively: TruHit, CoV-PPE_cs, MERS-PPE_cs.

I considered this interpretation in the repository when annotating.

**Comment by GemmaTuron** [comment]
Hi @Zainab-ik 

We did not discuss that, but I think Active Inactive is enough, we do not need to add the Active in Assay, as it is redundant
